spec/models/generic_work_spec.rb

```diff
@@ -4,14 +4,7 @@
 
 describe GenericWork do
   describe 'properties' do
-    it 'is open visibility by default.' do
-      expect(subject.visibility).to eq 'open'
-    end
 
-    it 'it cannot be set to anything other than open visibility.' do
-      subject.visibility='restricted'
-      expect(subject.visibility).to eq 'open'
-    end
```

Comment:
We need specs to replace the ones that are being removed. The command line execution path for creating a work by directly invoking the model, should result in open visibility.  

The controller should have a spec that to check that the visibility is private is when the draft checkbox was selected, and open otherwise.